docs-devsite/vertexai-preview.generationconfig.md

```diff
@@ -26,7 +26,8 @@ export interface GenerationConfig
 |  [frequencyPenalty](./vertexai-preview.generationconfig.md#generationconfigfrequencypenalty) | number |  |
 |  [maxOutputTokens](./vertexai-preview.generationconfig.md#generationconfigmaxoutputtokens) | number |  |
 |  [presencePenalty](./vertexai-preview.generationconfig.md#generationconfigpresencepenalty) | number |  |
-|  [responseMimeType](./vertexai-preview.generationconfig.md#generationconfigresponsemimetype) | string | Output response mimetype of the generated candidate text. Supported mimetypes are <code>text/plain</code> (default, text output) and <code>application/json</code> (JSON response in the candidates). The model needs to be prompted to output the appropriate response type, otherwise the behavior is undefined. This is a preview feature. |
+|  [responseMimeType](./vertexai-preview.generationconfig.md#generationconfigresponsemimetype) | string | Output response mimetype of the generated candidate text. Supported mimetypes are <code>text/plain</code> (default, text output) and <code>application/json</code> (JSON response in the candidates). |
```

Comment:
NOT BLOCKING

mimetype -> MIME type

(comment applies to several other instances, even in other files)

Reply:
Fixed.